fastlane/lib/fastlane/actions/danger.rb

```diff
@@ -11,9 +11,14 @@ def self.run(params)
 
         danger_id = params[:danger_id]
         dangerfile = params[:dangerfile]
+        base = params[:base]
+        head = params[:head]
         cmd << "--danger_id=#{danger_id}" if danger_id
         cmd << "--dangerfile=#{dangerfile}" if dangerfile
         cmd << "--fail-on-errors=true" if params[:fail_on_errors]
+        cmd << "--new-comment" if params[:new_comment]
+        cmd << "--base=#{base}" if base
```

Comment:
Could we try to keep this consistent by just checking `if params[:base]` and `if params[:head]`?